packages/checkout-ui-extensions/README.md

```diff
@@ -0,0 +1,38 @@
+# `@shopify/checkout-ui-extensions`
+
+This library contains type definitions and other utilities for writing UI extensions in Shopify’s checkout. However, nothing in this library is strictly required to write a UI extension. As a developer building a UI extension, you can forego the utilities this library provides entirely, and instead use the [global `shopify` API](documentation/globals.md) directly in a plain JavaScript file. The script below is a valid UI script without any additional processing:
+
+```js
+shopify.extend('Checkout::Feature::Render', (root, api) => {
+  const button = root.createComponent('Button', {
+    onPress() {
+      console.log('Upsold!');
```

Comment:
Can we remove this log? Feels a little gross to make this seem like a celebration :P

Reply:
Technically this example doesn't even work since `api.done()` is not available in checkout... Might want to just put a comment placeholder here or something.